### ISSUE TITLE
Discriminator and adversarial generation loss are not implemented in the code?

### ISSUE THREAD

**Issue by TimRuanwx** [BradyFU/DVG-Face#6]
论文中提到的判别器和对抗生成loss在代码中没有看到相应的实现？

```
# all losses
            loss = loss_ort + loss_pix + loss_ip
```

which is inconsistent with 

![temp](https://user-images.githubusercontent.com/51785266/200517161-c17f0402-a40e-4296-8c2c-9cd53acf3526.png)

**Comment by moliqingwa** [comment]
> 论文中提到的判别器和对抗生成loss在代码中没有看到相应的实现？
> 
> `# all losses loss = loss_ort + loss_pix + loss_ip`
> 
> which is inconsistent with
> 
> ![temp](https://user-images.githubusercontent.com/51785266/200517161-c17f0402-a40e-4296-8c2c-9cd53acf3526.png)

同问，我也没看到

**Comment by TimRuanwx** [comment]
![Screenshot from 2022-11-23 11-36-40](https://user-images.githubusercontent.com/51785266/203464677-edd00c18-e830-48e2-a581-0445a7d29de8.png)
这是用id_sample得到id解码出来的fake_vis和fake_nir。
仅有识别loss和相似度loss不能避免这种情况。
RUN！
